tableau_blueprints/job_status.py

```diff
@@ -0,0 +1,132 @@
+from httprequest_blueprints import execute_request
+
+import tableauserverclient as TSC
+import argparse
+import sys
+import os
+import pickle
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_UNKNOWN_ERROR = 3
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_RESOURCE = 201
+EXIT_CODE_FINAL_STATUS_ERRORED = 211
+EXIT_CODE_FINAL_STATUS_CANCELLED = 212
+EXIT_CODE_STATUS_INCOMPLETE = 210
+EXIT_CODE_JOB_NOT_FOUND = 404
+EXIT_CODE_JOB_CACNELLED = 403
+EXIT_CODE_GENERIC_QUERY_JOB_ERROR = 400
```

Comment:
404, 403, and 400 are not valid exit codes - they would result in some other value in the 0-255 range. Let's adjust to be contiguous with the rest of the error exit codes.